kuma/static/styles/components/codepen-iex/codepen-iex.scss

```diff
@@ -0,0 +1,21 @@
+@import '../../includes/vars';
+@import '../../includes/mixins';
+
+.codepen-iex {
+    position: relative;
+
+    .tryit {
+        position: absolute;
+        top: 10px;
+        right: 10px;
+        background-color: #fff;
+        border: 3px solid #3d7e9a;
+        height: 2rem;
+        font-size: 1rem;
+        cursor: pointer;
+
+        &:hover {
+            @include focus-visible();
+        }
+    }
+}
```

Comment:
Is the static-sample-code style already defined somewhere else?

Reply:
Yes, that is being taken care of sitewide via Prismjs